routes/utils.js

```diff
@@ -477,9 +477,11 @@ async function gen_mail_opt (options, variables)
     // replace variable in message
     for (let key in variables) {
         let value = variables[key];
+        if (!value) { value = '';} // value may not be defined
```

Comment:
Should test undefined/null rather that ! for possible (future) boolean values

Reply:
https://github.com/genouest/genouestaccountmanager/pull/171/commits/66f27bd7ae825cd057f12776748318eee7f328c2